addons/web/static/src/core/dialog/dialog.js

```diff
@@ -21,6 +21,9 @@ export class Dialog extends Component {
             { altIsOptional: true }
         );
         useSubEnv({ inDialog: true });
+        this.bodyTemplate = "web.DialogBodyDefault";
+        this.footerTemplate = "web.DialogFooterDefault";
```

Comment:
I would define them on the Class directly, what do you think? I think it makes more sense and it is easier to override